api/client.go

```diff
@@ -54,6 +55,9 @@ func (c *Client) GetLatestApplicableTag(id string) (*core.LatestTagResponse, err
 	if res.StatusCode < 200 || res.StatusCode >= 300 {
 		return nil, fmt.Errorf("status code is not in the expected range (%d), response body: %q", res.StatusCode, string(data))
 	}
+	if strings.TrimSpace(string(data)) == "null" {
```

Comment:
the API should change to send back an empty response instead of returning "null".

Reply:
do you mean JSON encoded string of `core.LatestTagResponse{}` as an empty response?

Reply:
yes, that is what i meant.